sc/managers/configloaders/kubernetes/kubeconnector/connector.go

```diff
@@ -0,0 +1,179 @@
+package kubeconnector
```

Comment:
No need to make a spearate direcotry for this. Put the connector in the `kube` directory itself